src/Microsoft.Azure.WebJobs.Extension.Dapr/Utils/JsonElementConverter.cs

```diff
@@ -0,0 +1,27 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.WebJobs.Extension.Dapr.Utils
+{
+    using System;
+    using System.Text.Json;
+    using System.Text.Json.Serialization;
+
+    internal class JsonElementConverter : JsonConverter<JsonElement>
```

Comment:
This is needed to be able to convert the JsonElement fields into string and vice-versa correctly. If not for this class, it will be serialized with `ToString()` instead of JsonSerializer. 

```c#
        /// <summary>
        /// Gets the payload of the pub/sub event.
        /// </summary>
        /// <remarks>
        /// The subscribers will receive this payload as the body of a Cloud Event envelope.
        /// </remarks>
        [JsonPropertyName("payload")]
        [JsonConverter(typeof(JsonUtils.JsonElementConverter))]
        public JsonElement Payload { get; }
```

Here is a small example - https://dotnetfiddle.net/8APcaH

```c#
JsonElement element = JsonDocument.Parse("{\"key1\": \n \"foo+bar\"}").RootElement;
JsonSerializerOptions options = new JsonSerializerOptions()
{
    Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping, 
};


Console.WriteLine(element.ToString());
Console.WriteLine(JsonSerializer.Serialize(element));
Console.WriteLine(JsonSerializer.Serialize(element, options));
```

Output:
```
{"key1": 
 "foo+bar"}
{"key1":"foo\u002Bbar"}
{"key1":"foo+bar"}
```